examples/llm/scripts/export_for_inference.py

```diff
@@ -0,0 +1,180 @@
+# Copyright 2022 MosaicML Examples authors
+# SPDX-License-Identifier: Apache-2.0
+
+import contextlib
+import os
+import sys
+import tempfile
+import warnings
+from urllib.parse import urlparse
+
+import torch
+from composer import Trainer
+from composer.utils import (get_device, maybe_create_object_store_from_uri,
+                            reproducibility)
+from omegaconf import OmegaConf as om
+
+from examples.llm import TorchCausalAttention
+from examples.llm.src.model_registry import COMPOSER_MODEL_REGISTRY
+
+
+def gen_random_batch(batch_size, cfg):
+    # generate input batch of random data
+    batch = {}
+    batch['input_ids'] = torch.randint(
+        low=0,
+        high=cfg.model.vocab_size,
+        size=(batch_size, cfg.max_seq_len),
+        dtype=torch.int64,
+    )
+    batch['attention_mask'] = torch.ones(size=(batch_size, cfg.max_seq_len),
+                                         dtype=torch.int64)
+    return batch
+
+
+def build_composer_model(cfg):
+    warnings.filterwarnings(
+        action='ignore',
+        message='Torchmetrics v0.9 introduced a new argument class property',
+    )
+    try:
+        return COMPOSER_MODEL_REGISTRY[cfg.name](cfg)
+    except:
+        raise ValueError(f'Not sure how to build model with name={cfg.name}')
+
+
+def main(cfg):
+    # Filter deprecation warning from torch internal usage
+    warnings.filterwarnings(
+        action='ignore',
+        category=UserWarning,
+        message=
+        f'torch.distributed.*_base is a private function and will be deprecated.*',
+    )
+
+    reproducibility.seed_all(cfg.get('seed', 17))
```

Comment:
using a default 17 seed reads like a magical number. Perhaps:
```
if cfg.get('seed', None) is not None:
    reproduciblity.seed_all(...)
```

Reply:
Actually this is not needed at all. Removed it.